### ISSUE TITLE
"Could not detect a running Discord client!" using WSL

### ISSUE THREAD

**Issue by yohanaff** [Almighty-Alpaca/Intellij-Discord-Integration#415]
I'm having this issue using the Discord Integration V2 on my IDE that is on WSL. I'm using Discord on my host machine, maybe that's the problem?

<img width="560" alt="discord-error" src="https://github.com/Almighty-Alpaca/JetBrains-Discord-Integration/assets/85047030/9ff61aae-ffce-4d71-a90e-32c3dd42096a">

**Comment by MoritzHinterleitner** [comment]
I don't think that discord and the ide can communicate through wsl. You can try something like https://github.com/OliverKovacs/wslrpc and see if it works

**Comment by yohanaff** [comment]
> I don't think that discord and the ide can communicate through wsl. You can try something like https://github.com/OliverKovacs/wslrpc and see if it works

Ok, I solved it by installing Discord on WSL :)
